modules/swoop/main.tf

```diff
@@ -6,12 +6,12 @@ module "swoop_namespace" {
   namespace             = var.namespace
 }
 
-module "swoop_api" {
+module "swoop_bundle" {
   count   = var.deploy_swoop_api == true ? 1 : 0
```

Comment:
should this also be changed to 'deploy_swoop_bundle' rather than 'deploy_swoop_api' ?

Reply:
No, in here since we only have swoop_api in the swoop_bundle, then the only trigger is `var.deploy_swoop_api == true`. However, when swoop-caboose and swoop-conductor become available I plan to change this trigger to count = var.deploy_swoop_api == true || var.deploy_swoop_caboose == true || var.deploy_swoop_conductor == true ? 1 : 0`

Reply:
Got it.